src/controllers/timeEntryController.js

```diff
@@ -365,10 +369,23 @@ const timeEntrycontroller = function (TimeEntry) {
     });
   };
 
+  const getCurrentDate = () => {
+    const currentDate = new Date();
+    const year = currentDate.getFullYear();
+    const month = String(currentDate.getMonth() + 1).padStart(2, "0");
+    const day = String(currentDate.getDate()).padStart(2, "0");
+    const formattedDate = `${year}-${month}-${day}`;
+
+    return formattedDate;
+  }
```

Comment:
```suggestion
```
Add this in a utils file if it's needed, but you can just use moment() as used elsewhere in this file.